README.md

```diff
@@ -4,6 +4,11 @@ Share shell spells.
 
```

Comment:
`Add testing and fix errors` seems kinda arbitrary.
How about one commit like `Add basic test for readSecondToLastLine` and commit(s) before where you fix the bugs?

Reply:
Can we start caring about commit hygiene after the initial bootstraping (this PR)?